### DISCUSSION TITLE
Spaceship published version is 4 years old and does not include passbook

### DISCUSSION THREAD

**Discussion by pkpfr** [fastlane/fastlane#17389] [Q&A]
Importing `gem spaceship` results in an error when trying to use Passbook, since this was added after v1.0.0.

How can I import spaceship in a ruby project using the current code?

**Comment by rogerluan** [comment]
I just import the entire `gem "fastlane"` 😞 I agree it's not ideal, but it works for me.
Then `require spaceship`

**Comment by Jasminol** [comment]
There are nine departments in the base RuboCop gem. Numbers might change in time and are given here only for the rough overview of what the gem offers.

Layout - 87 cops
Linting - 81 cops
Metrics - 10 cops
Migration - 1 cop
Naming - 16 cops
Security - 5 cops
Style Cops - 170 cops
Bundler - 4 cops
Gemspec - 4 cops    [My Indigo Card](https://www.myindigocard.us/)
